### DISCUSSION TITLE
How to call other unix files within wails

### DISCUSSION THREAD

**Discussion by sgx-wb** [wailsapp/wails#4199] [Q&A]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

I created an initial wails project with only the following modifications
![image](https://github.com/wailsapp/wails/assets/96750945/26630b16-1e70-42b6-a54a-62477d6c670e)
![image](https://github.com/wailsapp/wails/assets/96750945/dd0a8336-e849-459d-a742-280fc5c7f64b)
The unix file I want to use is https://github.com/yt-dlp/yt-dlp/releases 
![image](https://github.com/wailsapp/wails/assets/96750945/54725ae3-973f-4094-8cd1-c2ed4e89e23c)
Normally, the result of the call should be displayed, but now it shows that the unix file cannot be found. According to the file size, the unix file should have been built into the wails.
![image](https://github.com/wailsapp/wails/assets/96750945/2d66ef95-51d2-48bf-ae74-958b64e7f38b)

### Self-service

- [ ] I'd be willing to address this documentation request myself.

**Comment by sgx-wb** [comment]
[myWailsTest.zip](https://github.com/wailsapp/wails/files/13348677/myWailsTest.zip)
I packed the code. Can someone test it for me? the yt-dlp in static is the unix of mac. If you are using windows, you need to replace it.

**Comment by stffabi** [comment]
The embed assets `fs.FS` has no relation with what is getting packaged with your app to be stored on the local filesystem. The embed FS is a virtual filesystem in your application and is used to store files for the frontend website. If you want to execute a binary that is in your embed FS you need to first write that file to your local filesystem or package it with your app with a NSIS installer or bundling it into your macOS .app file after building has been done.

**Comment by sgx-wb** [comment]
@stffabi Hello, thank you for your reply. It seems that if I use wails build to generate an application and embed a Unix file, if I want to execute this Unix file on another computer, do I need to first write this Unix file locally and then call the Unix file?
